docs/guides/hedgehog-engine/blueblur/levels/importing-collision/importing-collision.md

```diff
@@ -0,0 +1,48 @@
+---
+description: Importing Collision tutorial for Sonic Generations
+---
+!!! info
+    This guide uses the following tools:
+
+    - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/){ target="_blank"}
+	- [Blender (You can use any other 3D editor)](https://blender.org){ target="_blank"}
```

Comment:
This isn't correctly indented. Sonic GLvl and Blender need to be at the same indentation level